internal/app/connectconformance/connectconformance.go

```diff
@@ -204,27 +207,39 @@ func run( //nolint:gocyclo
 	}
 
 	filter := newFilter(run, skip)
+	var filteredTestCount int
+	type serverConfig struct {
+		serverInstance
+		isGrpcClient, isGrpcServer bool
+	}
```

Comment:
This is for two things:
1. It fixes the reporting of the number of server configurations, by including the gRPC implementations as separate configurations.
2. We now need to know the total number of test cases (after filtering), even when we're not using verbose printing. This is so that the results know if there were any test cases skipped or not run (due to client hanging/crashing).